homeassistant/components/russound_rio/quality_scale.yaml

```diff
@@ -18,8 +18,8 @@ rules:
     comment: |
       This integration does not provide additional actions.
   docs-high-level-description: done
```

Comment:
Another note, let's make it "The `russound_rio` integration allows" -> "The Russound RIO integration allows"

Reply:
Fixed in https://github.com/home-assistant/home-assistant.io/pull/36407